redis/client.go

```diff
@@ -0,0 +1,219 @@
+package redis
+
+import (
+	"context"
+	"fmt"
+	"github.com/DataDog/datadog-go/statsd"
+	"github.com/coinbase/memcachedbetween/pool"
+	"github.com/coinbase/mongobetween/util"
+	"github.com/coinbase/redisbetween/utils"
+	"go.uber.org/zap"
```

Comment:
Nit: Pull down `github.com` and `go.uber.org` imports to separate group

Reply:
done